### ISSUE TITLE
Suggestions not working

### ISSUE THREAD

**Issue by Andrew-Sor** [Goodwy/Keyboard#2]
Hello. I found that the prompts are not working in your keyboard. As far as I know, they also don't work in FlorisBoard. When typing the word "he", the hints are displayed "he0", "he1", "he3", etc. For the Russian language, they are not displayed at all. Are there any ideas to solve this problem?

**Comment by Maxitendo1** [comment]
It could be possible to use this repo which has already swipe to type, voice input, autocorrect, predictive text and local AI. However it is written in C++ so it probably won't work, because Inkwell is written in Kotlin.

https://github.com/futo-org/android-keyboard

**Comment by Andrew-Sor** [comment]
I've read in other similar issues that you need to build an application from source code somehow. Can I find out exactly what I need to do for this?

**Comment by Maxitendo1** [comment]
I don't have experience in coding, but as Inkwell is based on FlorisBoard, this answer might work for you.

1. In short, you clone the repository with powershell or maybe:

```git clone https://github.com/goodwy/inkwell.git```

2. Set up the build environment, install Android SDK and Gradle.

3. Configure the build, go to the inkwell directory, copy the build.gradle file from root directory. Overwrite the default build.gradle.

4. If you're done, then build and package with

```./gradlew assembleRelease```

Full AI answer:
 https://search.brave.com/search?q=how+to+build+florisboard&source=android&summary=1

**Comment by Maxitendo1** [comment]
And for integrating C++ and Kotlin, it could work if you build a "bridge" between them. Don't know if it would really work:

To use a C++ library in a Kotlin Multiplatform Mobile (KMM) app, you'll need to create a bridge between the two languages. Here's a step-by-step approach:
 https://search.brave.com/search?q=can+you+build+kotlin+and+c%2B%2B+in+one+app&source=web&summary=1

**Comment by Andrew-Sor** [comment]
No, AI is bad idea. I'll probably ask the one who managed to fix the assembly hints what they were doing.
